wremnants/OutputTools.py

```diff
@@ -16,3 +16,17 @@ def fillTemplatedFile(templateFile, outFile, templateDict, append=False):
         outFile.write(result)
 
 
+def resetBinError(histo):
```

Comment:
Well maybe @bendavid should confirm that if you write a hist with storage=Double using hist_to_root that it will have no error. Alternatively you could use storage=Weight and then data=np.stack((origin_hist.values(flow=True), np.zeros_like(origin_hist.variances(flow=True)), axis=-1)

Reply:
no, indeed converting a histogram of storage type double will create ROOT histogram which does NOT have sumw2() called (and therefore the errors default to sqrt(bin content))

So you need to explicitly zero the errors before conversion.

can just do
```python
hist_no_error = hist.copy()
hist_no_error.variances(flow=True)[...] = 0.
```

Reply:
Removed now, no longer needed indeed